source/elements/oneTBB/source/deprecated/task_arena_attach_tag.rst

```diff
@@ -0,0 +1,64 @@
+.. SPDX-FileCopyrightText: 2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+==================
+task_arena::attach 
+==================
+**[deprecated.task_arena_attach_tag]**
+
+    .. caution::
+
+        Deprecated in oneTBB Specification 1.1.
+
+A set of methods for constructing a ``task_arena`` with ``attach``.
```

Comment:
```suggestion
A set of methods for constructing attached ``task_arena``.
```

Consider this wording. If you like it, then consider also applying it in other places.

Reply:
We have decided to leave wording as is.

Reply:
Discussed offline, decided to keep the wording from the old spec.